lsp-mode.el

```diff
@@ -3638,16 +3661,19 @@ disappearing, unset all the variables related to it."
     (-let* ((created-watches (lsp-session-watches (lsp-session)))
             (root-folders (cl-set-difference
                            (lsp-find-roots-for-workspace lsp--cur-workspace (lsp-session))
-                           (ht-keys created-watches))))
+                           (ht-keys created-watches)))
+            (workspace-root (lsp--workspace-root lsp--cur-workspace))
+            (workspace-sub-folders (lsp--workspace-watch-folders lsp--cur-workspace)))
       ;; create watch for each root folder without such
       (dolist (folder root-folders)
         (let ((watch (make-lsp-watch :root-directory folder)))
           (puthash folder watch created-watches)
           (lsp-watch-root-folder (file-truename folder)
+                                 (when (f-equal? folder workspace-root)
```

Comment:
Shouldn't this to use `lsp-f-same?`?
I don't think we need to follow symlink when compare the `workspace-root` here

Reply:
didn't know about that one. looks good to me.